runtime/THREADS.md

```diff
@@ -1,12 +1,12 @@
 # Threads
 
-The following **8 threads** are used by the runtime:
+The following **26 threads** are used by the runtime:
```

Comment:
What is the reason for this increase (e.g. are there any performance issues in production)?

Reply:
Better performance for confidential client nodes that need to handle a lot of queries.